app/controllers/claims_controller.rb

```diff
@@ -37,26 +36,7 @@ def show
       return
     end
 
-    if params[:slug] == "select-home-address" && postcode
-      session[:claim_postcode] = postcode
-      session[:claim_address_line_1] = params.dig(:claim, :address_line_1)
-      if address_data.nil?
-        redirect_to claim_path(current_journey_routing_name, "no-address-found") and return
-      else
-        # otherwise it takes you to "no-address-found" on the backlink from the slug sequence
-        @backlink_path = claim_path(current_journey_routing_name, "postcode-search")
```

Comment:
I _think_ this backlink path logic might be missing now.

Reply:
yep! if it's not just a backlink to the previous slug i.e. there's some logic behind it, we override the `backlink_path` in the base form 👍